test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -300,7 +292,7 @@ private static async Task CreateContentZip(string contentRoot, string zipPath, p
 
         private static async Task<Uri> CreateBlobSas(string filePath, string blobContainer, string blobName)
         {
-            string connectionString = Environment.GetEnvironmentVariable(ConnectionStringNames.Storage);
+            string connectionString = Environment.GetEnvironmentVariable("AzureWebJobsStorage");
```

Comment:
The connection string pattern is using an extension method on `IConfiguration` (essentially replacing the connection string provider)

Reply:
Yes, I will fix this as well. It's part of the Linux test that's currently disabled. I was just quickly trying to figure out how far I could get before it failed.